### DISCUSSION TITLE
Black Screen

### DISCUSSION THREAD

**Discussion by bbday** [cefsharp/CefSharp#3712] [General]
I have sometimes, not always, black screen with OffScreen > ScreenshotOrNull()

What is reason of blackscreen and config/setup can trigger it?

ScreenshotAsync() < return old screen (also after new page loaded)

**Comment by amaitland** [comment]
Check the log file for errors. 

If the website rendered a black screen then that's what you'll get.

By default ScreenshotAsync will return an existing screenshot. I'd suggest you look at the source.

Long term I'd suggest migrating to headless chrome with puppateer Sharp or similar.

**Reply by bbday** [reply]
thanks for reply i'll enable log error for cef i hope i'll find answer
